### PR TITLE
updated diumoo (1.5.0)

### PR DESCRIPTION
**Pull request by dejavuzhang**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/diumoo.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'diumoo' do
-  version :latest
-  sha256 :no_check
+  version '1.5.0'
+  sha256 'f6a6628dab6c64752f6827e453368c18dcd06962ee0823d51d51c439cf7e04b6'
 
-  url 'http://share.diumoo.net/static/diumoo.zip'
+  url 'https://github.com/shanzi/diumoo/releases/download/v1.5.0/diumoo.zip'
```
Comment (vitorgalvao) [inline Casks/diumoo.rb]: Needs a `# github.com/shanzi/diumoo was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/diumoo.rb]: `url "https://github.com/shanzi/diumoo/releases/download/v#{version}/diumoo.zip"`

Comment (vitorgalvao) [inline Casks/diumoo.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by vitorgalvao** [comment]
Did you really run the `brew cask style` command? Because the build is saying you did not.

**Comment by dejavuzhang** [comment]
Are you referring to what is written in CONTRIBUTING.md? If so, yes I did.
Thanks.

**Comment by vitorgalvao** [comment]
No, I’m referring to the last item of the checklist in your top post.

**Comment by dejavuzhang** [comment]
I followed the instruction listed in CONTRIBUTING.md, so there should not be the problem…

**Comment by vitorgalvao** [comment]
Make the changed I requested, and it’ll be fixed.

File: Casks/diumoo.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
 cask 'diumoo' do
-  version :latest
-  sha256 :no_check
+  version '1.5.0'
+  sha256 'f6a6628dab6c64752f6827e453368c18dcd06962ee0823d51d51c439cf7e04b6'
 
-  url 'http://share.diumoo.net/static/diumoo.zip'
+  # github.com/shanzi/diumoo was verified as official when first introduced to the cask
+  url 'https://github.com/shanzi/diumoo/releases/download/v#{version}/diumoo.zip'
```
Comment (vitorgalvao) [inline Casks/diumoo.rb]: Need to use double quotes here, for it to work. After changes, you need to rerun the commands.

File: Casks/diumoo.rb [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
-  version :latest
-  sha256 :no_check
+  version '1.5.0'
+  sha256 'f6a6628dab6c64752f6827e453368c18dcd06962ee0823d51d51c439cf7e04b6'
 
-  url 'http://share.diumoo.net/static/diumoo.zip'
+  # github.com/shanzi/diumoo was verified as official when first introduced to the cask
+  url "https://github.com/shanzi/diumoo/releases/download/v#{version}/diumoo.zip"
+  appcast 'https://github.com/diumoo/diumoo.github.com/releases.atom',
+          checkpoint: 'cf1c64c1e44fcbc61b32c61dbafb347ea7a1911d493f0cdca8a8a3911def29cf'
```
Comment (vitorgalvao) [inline Casks/diumoo.rb]: Value is wrong. Should be `cf36bbd20b99d8fd4f5c6ceaa95a2b8819ef1f5a15f05e35e8037ac8d0aaa0b3`.

**Comment by vitorgalvao** [comment]
Looks good. Thank you.

**Pull request merged**